pyxrf/model/command_tools.py

```diff
@@ -112,6 +126,15 @@ def fit_pixel_data_and_save(working_directory, file_name, *,
         # update incident energy, required for XANES
         if incident_energy is not None:
             param_sum['coherent_sct_energy']['value'] = incident_energy
+            print(f"Using incident beam energy passed as the function parameter.")
```

Comment:
I removed f-string. The energy value is printed below. Actually the prints will soon be replaced by the logger output, since logger now enabled in command line functions.